File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>



void leftTower(int height);

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    } while (height < 1 || height > 8);
    leftTower(height);
}


void leftTower(int height)
{
    int height = get_int("enter : ");
    for (int i = height - 1; i >= 0; i--)
    {
        for (int j = i; j > 0; j--)
        {
            printf(" ");
        }
        printf("#");
    }
    return 0;
}

File: week6/sentimental-mario-more/mario.py
from cs50 import get_int


def main():
    height = get_int('Height: \n')
    while(height < 1 or height > 8):
        height = get_int('Height: \n')
    for i in range(height):
        gap = ' ' * (height - 1 - i)
        tower = '#' * (i+1)
        print(f'{gap}{tower}  {tower}')

main()

